src/Bridge/Symfony/Translator/Label.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Symfony\Translator;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class Label
```

Comment:
Let's name it as an action and make it invokable
```suggestion
final class GetLabel
```